src/Peering/Peering/Az.Peering.psd1

```diff
@@ -53,7 +53,7 @@ DotNetFrameworkVersion = '4.7.2'
 # ProcessorArchitecture = ''
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '1.5.0'; })
+RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '1.5.1'; })
```

Comment:
@rileymckenna this should be bumped up to 1.5.2

Reply:
@cormacpayne should i increment the version of the package to 0.1.1?